calgen/models/Calendar.py

```diff
@@ -53,7 +53,7 @@ def to_ics(self):
             'description': self.description,
             'dtstamp': datetime.now(),
             'class': 'public',
-            'transp': 'opaque' if self.calendar_type == CalendarTypes.NATIONAL else 'transparent',
+            'transp': 'OPAQUE' if self.calendar_type == CalendarTypes.NATIONAL else 'TRANSPARENT',
```

Comment:
TIL about representing availability in .ics files :)

Reply:
YOU NEED TO YELL ABOUT IT THOUGH. Otherwise it doesn't count 🤷